src/lib/y2storage/proposal/partition_creator.rb

```diff
@@ -53,10 +53,11 @@ def create_partitions(distribution)
         self.devicegraph = original_graph.duplicate
 
         distribution.spaces.each do |space|
-          type = space.partition_type
           vols = space.volumes
           disk_space = space.disk_space
-          process_space(vols, disk_space, type)
+          num_logical = space.num_logical
+          usable_size = space.usable_size
+          process_space(vols, disk_space, usable_size, num_logical)
```

Comment:
Why not simply pass space.usable_size and space.num_logical here and be two lines more concise?
The additional variables don't contribute very much to readability IMHO, and they don't seem to be used anywhere else.